.github/workflows/main.yml

```diff
@@ -5,8 +5,7 @@ jobs:
     strategy:
       matrix:
         ruby-version:
-          - "2.7"
-          - "3.0"
+          - "3.1"
```

Comment:
✅ Agree. I don't think there are reasons to keep 2.7 / 3.0, while it will fail otherwise 👍

Reply:
👍